test/sequential/xyk-pallet.api.maintenance-config.test.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ *
+ * @group sequential
```

Comment:
let's run those tests separately in some other group, like "maintenance". I think that running this on seq could cause some issues if the maintenance mode is left enabled for some reason.

Reply:
sure, changed